examples/Bremsstrahlung/include/simulation_defines/param/bremsstrahlung.param

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2016-2017 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace bremsstrahlung
+{
+
+/** enable bremsstrahlung */
+#ifndef ENABLE_BREMSSTRAHLUNG
+#define ENABLE_BREMSSTRAHLUNG 0
+#endif
+
+/** params related to the energy loss and deflection of the incident electron
+ */
+namespace electron
+{
+    /** Minimal kinectic electron energy in MeV for the lookup table.
+     * For electrons below this value Bremsstrahlung is not taken into account.
+     */
+    constexpr float_64 MIN_ENERGY_MeV = 0.5;
+
+    /** Maximal kinetic electron energy in MeV for the lookup table.
+     * Electrons above this value cause a out-of-bounds access at the
+     * lookup table. Bounds checking is enabled for "CRITICAL" log level.
+     */
+    constexpr float_64 MAX_ENERGY_MeV = 100.0;
+
+    /** Minimal polar deflection angle due to screening. */
+    constexpr float_64 MIN_THETA = 0.001;
+
+    /** number of lookup table divisions for the kappa axis.
+     * Kappa is the energy loss normalized to the initial kinetic energy.
+     * The axis is scaled linearly.
+     */
+    constexpr uint32_t NUM_SAMPLES_KAPPA = 64;
+
+    /** number of lookup table divisions for the initial kinetic energy axis.
+     * The axis is scaled logarithmically.
+     */
+    constexpr uint32_t NUM_SAMPLES_EKIN = 128;
+
+    /** Kappa is the energy loss normalized to the initial kinetic energy.
+     * This minimal value is needed by the numerics to avoid a division by zero.
+     */
+    constexpr float_64 MIN_KAPPA = 1.0e-10;
+
+} // namespace electron
+
+/** params related to the creation and the emission angle of the photon
+ */
+namespace photon
+{
+    /** Low-energy threshold in keV of the incident electron for the creation of photons.
+     * Below this value photon emission is neglected.
+     */
+    constexpr float_64 SOFT_PHOTONS_CUTOFF_keV = 5.0;
+
+    /** number of lookup table divisions for the delta axis.
```

Comment:
There must be a empty line after the short `brief`. This should be changed in all documantions